tests/RemoveAccentsTest.elm

```diff
@@ -0,0 +1,19 @@
+module RemoveAccentsTest exposing (removeAccentsTest)
+
+import Expect
+import String.Extra exposing (removeAccents)
+import Test exposing (..)
+
+
+removeAccentsTest : Test
+removeAccentsTest =
+    describe "removeAccents"
+        [ test "Should result string without acents" <|
+            \() ->
+                removeAccents "áàãâäéèêëíìîïóòõôöúùûüçÁÀÃÂÄÉÈÊËÍÌÎÏÓÒÕÖÔÚÙÛÜÇ"
+                    |> Expect.equal "aaaaaeeeeiiiiooooouuuucAAAAAEEEEIIIIOOOOOUUUUC"
+        , test "Should result in string without acents" <|
```

Comment:
Latest versions of elm-test require that each `test` is given a unique string.

Also, `acents` -> `accents`